src/tool/hpcrun/scripts/hpcrun.in

```diff
@@ -503,10 +508,21 @@ fi
 ulimit -S -c unlimited >/dev/null 2>&1
 
 # laks 2016.10.06: add perfmon directory
-hpc_ld_library_path="${hpcrun_dir}:${papi_libdir}:${ext_dir}:${perfmon_libdir}:${roctracer_lib}"
+hpc_ld_library_path="${hpcrun_dir}"
+hpc_ld_library_path="${hpc_ld_library_path}${papi_libdir:+:${papi_libdir}}"
+hpc_ld_library_path="${hpc_ld_library_path}${ext_dir:+:${ext_dir}}"
+hpc_ld_library_path="${hpc_ld_library_path}${perfmon_libdir:+:${perfmon_libdir}}"
+hpc_ld_library_path="${hpc_ld_library_path}${roctracer_lib:+:${roctracer_lib}}"
 preload_list="${libmonitor_dir}/libmonitor.so ${preload_list}"
 preload_list="${hpcrun_dir}/libhpcrun.so ${preload_list}"
 
+# Set up the auditor or fake auditor, depending on what's configured.
+if test "$HPCRUN_AUDIT_USE_FAKE_AUDITOR" = 1 ; then
```

Comment:
I prefer the shorter HPCRUN_FAKE_AUDITOR (here and in the code)

Reply:
Done.